src/main/java/org/spongepowered/api/text/channel/MessageReceiver.java

```diff
@@ -135,4 +127,18 @@ default Locale getLocale() {
         return Locales.DEFAULT;
     }
 
+    /**
+     * Represents a {@link MessageReceiver} that can have their
+     * {@link MessageChannel} changed.
+     */
+    interface Mutable extends MessageReceiver {
+
+        /**
+         * Sets the message channel that messages sent by this source should be sent
```

Comment:
So what does this mean exactly? Is this effectively removing the need for events on a channel to have a player's message forwarded to other channels?